waflyctl.go

```diff
@@ -2434,10 +2434,10 @@ func main() {
 		//validate the config
 		validateVersion(*client, *serviceID, latest.Number)
 		Info.Println("Completed")
-		os.Exit(1)
+		os.Exit(0)
 	} else {
 		Info.Println("Nothing to do. Exiting")
-		os.Exit(1)
+		os.Exit(0)
```

Comment:
This is actually an error and should exit with 1.

Reply:
Ah, considering the above msg outputs to the Info log, rather than error. I didn't read this as an error case.

Reply:
But sure, I can change this. I do agree that it should probably be an error.
Should I tweak the log line?

Reply:
Good idea. 👍